src/main/java/edu/ucla/library/iiif/fester/verticles/ManifestVerticle.java

```diff
@@ -642,7 +638,8 @@ private Collection getCollection(final String[] aRow, final CsvHeaders aHeaders)
             final String label = StringUtils.trimToNull(aRow[aHeaders.getTitleIndex()]);
 
             if (label != null) {
-                return new Collection(IDUtils.encode(myHost, Constants.COLLECTIONS_PATH, id), label);
+                return new Collection(
+                        IDUtils.getResourceURI(myHost, IDUtils.getCollectionS3Key(id)).toString(), label);
```

Comment:
At the point where the line is wrapping because of nested methods, I tend to think it's more readable to break it out. So, for instance, put `IDUtils.getCollectionS3Key` into a `s3Key` (or something) variable and pass that variable to `IDUtils.getResourceURI`.